pydrive2/auth.py

```diff
@@ -147,20 +65,6 @@ class GoogleAuth(ApiAttributeMixin):
     and authorization.
     """
 
-    DEFAULT_SETTINGS = {
```

Comment:
Q: why do we move the outside the class? (it will break some project that rely on this to be inside)

Reply:
I didn't really expect anybody to use this outside of the GoogleAuth constructor. I can definitely add it back as a class static variable

Reply:
We do use this in `dvc-objects`

https://github.com/iterative/dvc-objects/blob/3bc5625dceaf7698e064074ec3b4c2b839b92335/src/dvc_objects/fs/implementations/gdrive.py#L208-L223

Reply:
we don't use it anymore in DVC, still let's not move things in this PR, let's focus on the migration first, please

Reply:
I've reverted the DEFAULT_SETTINGS to a class static variable